package.json

```diff
@@ -57,6 +57,7 @@
     "less": "2.7.1",
     "less-loader": "2.2.3",
     "postcss-loader": "0.9.1",
+    "prop-types": "^15.5.10",
```

Comment:
Should the `prop-types` package be a peer dependency?  There's been an ongoing debate about peer dependencies but at least for now I think it makes more sense for it to be a peer dependency since react is also one.